source/includes/administration/_organizations.md

```diff
@@ -195,6 +199,7 @@ Optional | &nbsp;
 ---- | ----
 `serviceConnections`<br/>Array[[ServiceConnection](#administration-service-connections)] | A list of service connections for which the organization may provision resources.<br/>*required :*`id`
 `parent`<br/>[Organization](#administration-organization) | The organization that will be the parent of the new organization. By default, it will default to the caller's organization.<br/>*required :*`id`
+`billingDay`<br/>*int* | The billing day of the organization. Must be between 1 and 28, the default value is 1.
```

Comment:
might be worth saying the 28th is inclusive.

Reply:
fa0e5b68c5f3b652a9a7f43781e4d0eb12061e16